apps/common/lib/lexical/ast/analysis/import.ex

```diff
@@ -76,4 +89,14 @@ defmodule Lexical.Ast.Analysis.Import do
 
   defp expand_function_keywords(_ignored, acc),
     do: acc
+
+  defp range_for_import(%Document{} = document, quoted) do
+    case Ast.Range.fetch(quoted, document) do
```

Comment:
Let's add `Ast.Range.get(range, document, default \\ nil)` and then we can use that directly.